tests/storage_test.go

```diff
@@ -253,5 +253,53 @@ var _ = Describe("Storage", func() {
 				close(done)
 			}, 400)
 		})
+
+		Context("With VM with two PVCs", func() {
```

Comment:
I do not understand. It is directly under the Discribe.

Reply:
I meant that you can put your context under describe on the line https://github.com/kubevirt/kubevirt/pull/843/files#diff-1a5e99cccdb08e6a6453c478e667f657R118, so the structure will be
```
Describe("Starting a VM", func() {
...
    Context("with two PVCs", func() {
        ...
        It("should succeed multiply times", func{ ... })
    })
```

Reply:
I understand now. This would result in more changes than necessary. Is this a blocker? Otherwise I would pretty much leave it here. When the test runs, the structure is nice and readable as you requested:
```
Starting a VM
  with two PVCs
    should succeed multiply times
```

Reply:
If it looks that way it good enough for me